web/js/map/natural-events/util.js

```diff
@@ -15,8 +15,18 @@ export const getTrackPoint = function(proj, clusterPoint, isSelected, callback)
   const overlayEl = document.createElement('div');
   const circleEl = document.createElement('div');
   const textEl = document.createElement('span');
-  const { properties } = clusterPoint;
+  const { properties, geometry } = clusterPoint;
   const content = document.createTextNode(formatDisplayDate(properties.date));
+
+  const { magnitudeValue, magnitudeUnit } = geometry;
+  const magnitudeContent = document.createElement('div');
+  const superscriptUnit = document.createElement('sup');
+  superscriptUnit.append('2');
+  const formattedMagnitudeUnit = magnitudeUnit === 'kts' ? ' kts' : ' NM';
+  magnitudeContent.append(magnitudeValue.toLocaleString());
```

Comment:
Be careful calling this directly on something we got from an API since it assumes that there is a value there.  If there is not, we get an error.  For example, [this event seems to cause an issue: ](http://localhost:3000/?v=-44.0328515625,-66.98218750000001,-28.6871484375,-43.4978125&e=EONET_5210,2021-01-15&efs=true&efd=2016-10-21,2022-02-18&efc=seaLakeIce&l=MODIS_Aqua_Sea_Ice(hidden),MODIS_Terra_Sea_Ice(hidden),VIIRS_SNPP_Brightness_Temp_BandI5_Night(hidden),VIIRS_SNPP_Brightness_Temp_BandI5_Day(hidden),VIIRS_NOAA20_Brightness_Temp_BandI5_Night(hidden),VIIRS_NOAA20_Brightness_Temp_BandI5_Day(hidden),MODIS_Aqua_Brightness_Temp_Band31_Night(hidden),MODIS_Aqua_Brightness_Temp_Band31_Day(hidden),MODIS_Terra_Brightness_Temp_Band31_Night(hidden),MODIS_Terra_Brightness_Temp_Band31_Day(hidden),IMERG_Precipitation_Rate(hidden),VIIRS_SNPP_DayNightBand_ENCC(hidden),VIIRS_SNPP_DayNightBand_At_Sensor_Radiance(hidden),VIIRS_SNPP_DayNightBand_AtSensor_M15(hidden),Reference_Labels_15m,Reference_Features_15m,Coastlines_15m(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2021-01-15-T00%3A00%3A00Z)

![Screen Shot 2022-02-18 at 10 34 38 AM](https://user-images.githubusercontent.com/1480000/154733607-74981f99-d6ee-4f3c-8135-ccb14f46a90e.png)